### ISSUE TITLE
Can't find velocity in window and window.jQuery when using optimizer of RequireJS

### ISSUE THREAD

**Issue by PeachScript** [cgwyllie/angular-velocity#20]
I use the following library by RequireJS. The Fragment of require config file:

```
...
paths: {
    'velocity': 'bower_components/velocity/velocity',
    'velocity.ui': 'bower_components/velocity/velocity.ui',
    'ngVelocity': 'bower_components/angular-velocity/angular-velocity'
}
...
shim: {
    'velocity': {
      deps: ['jQuery']
    },
    'velocity.ui': {
      deps: ['velocity']
    },
    'ngVelocity': {
      deps: ['velocity', 'velocity.ui', 'angular', 'ngAnimate']
    }
...
```

And I got the following error in browser when I used the r.js optimize the javascript files:
```
Uncaught Velocity and Velocity UI Pack plugin required, please include the relevant JS files. Get Velocity with: bower install velocity
```

Could you tell me how to use angular-velocity with RequireJS? Thank you :D

**Comment by cgwyllie** [comment]
Hi!

I don't have any experience with using RequireJS, but let's have a go!

Could you replace the angular-velocity.js contents with [this gist](https://gist.github.com/cgwyllie/b043d43656f64fe4770c) and let me know if it resolves the issue? If so, I'll look at tidying it up and releasing an update.

Thanks!

**Comment by PeachScript** [comment]
Thanks for your reply !

I'm not sure the problem be caused by me or caused by the angular-velocity.js,  I try to fixed it before, but no progress. However, I will try to fixed it next, and tell you at here if I have any problem. Maybe it just because of I use the error way for RequireJS. :D

**Comment by PeachScript** [comment]
Hi friend, I solve the problem just now. As I said, the problem be caused by me.

My configuration file is correct for RequireJS, but I use the r.js to compress code then I get the problem. In the docs of r.js, I found the configuration of `shim` has a option 'wrapShim', and the problem has been fixed when I set the `wrapShim` as true. The configuration file just like this:
```
requirejs.config({
  ...
  paths:{
    ...
  },
  shim: {
    ...
  },
  wrapShim: true
});
```
